src/main/groovy/org/pih/warehouse/monitoring/GrailsHttpTransactionNameProvider.groovy

```diff
@@ -0,0 +1,23 @@
+package org.pih.warehouse.monitoring
+
+import io.sentry.protocol.TransactionNameSource
+import io.sentry.spring.tracing.TransactionNameProvider
+import javax.servlet.http.HttpServletRequest
+
+/**
+ * Defines how Sentry transaction names for HTTP requests should be structured.
+ */
+class GrailsHttpTransactionNameProvider implements TransactionNameProvider {
+
+    String provideTransactionName(HttpServletRequest var1) {
```

Comment:
Could you change the `var1` to `request`? Uness there's something I'm missing the var1 had me double-take a few times during the review of this code.

Reply:
I used `var1` because that's what the abstract method in TransactionNameProvider uses and I remember you previously asked me to maintain method arg names from the interface. I agree that `request` is clearer though so I can change it

Reply:
Ugh. Yes, I did. :) 

I assumed the `arg1` was the default arg name provided by Intellij, but I guess that makes sense.